src/megatron/bridge/models/gemma/gemma_provider.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import dataclass
+from typing import Callable
+
+import torch
+from megatron.core import parallel_state
+from megatron.core.models.gpt import GPTModel as MCoreGPTModel
+from megatron.core.transformer.enums import AttnBackend
+
+from megatron.bridge.models.activations import openai_gelu
+from megatron.bridge.models.gpt_provider import GPTModelProvider
+
+
+@dataclass
+class GemmaModelProvider(GPTModelProvider):
+    """Configuration class for Gemma models."""
+
+    # configs that are common across model sizes
+    normalization: str = "RMSNorm"
+    activation_func: Callable = openai_gelu
+    gated_linear_unit: bool = True
+    position_embedding_type: str = "rope"
+    add_bias_linear: bool = False
+    seq_length: int = 8192
+    kv_channels: int = 256
+    attention_dropout: float = 0.0
+    hidden_dropout: float = 0.0
+    share_embeddings_and_output_weights: bool = True
+    # Note: different behavior compared to NeMo 1.0
```

Comment:
I wonder if we still needs this comment in mbridge since nemo1 is quite a long time ago